### PR TITLE
Add some Purdue faculty

### PR DESCRIPTION
**Pull request by GiantSeaweed**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

_Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [X] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [X] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [X] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [X] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -20079,6 +20087,7 @@ Xiaojun Shen,University of Missouri - Kansas City,http://sce2.umkc.edu/csee/shen
 Xiaojun Wan 0001,Peking University,http://www.icst.pku.edu.cn/lcwm/wanxj,lTTeBdkAAAAJ
 Xiaojun Ye,Tsinghua University,http://www.thss.tsinghua.edu.cn/publish/soften/3131/2010/20101219111626359565665/20101219111626359565665_.html,NOSCHOLARPAGE
 Xiaojun Zhai,University of Essex,https://www.essex.ac.uk/people/zhaix77307/xiaojun-zhai,sLYyja0AAAAJ
+Xiaokang Qiu,Purdue University,https://engineering.purdue.edu/~xqiu/,xGHVK6sAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Not on the list of cross-appointed faculty in CS. https://www.cs.purdue.edu/people/faculty/index.html

**Review by emeryberger — CHANGES_REQUESTED** [review]
You also need to restore the alias for Tamal Dey (both must be listed).

File: csrankings.csv (hunk 1 above)
Comment (GiantSeaweed) [reply csrankings.csv]: I have deleted his entry.

**Comment by GiantSeaweed** [comment]
I have restored the alias for Tamal Dey. Thanks.

**Review by emeryberger — APPROVED** [review]

**Comment by emeryberger** [comment]
Apparently there are conflicts with `csrankings.csv`. Please resolve - thanks.

**Comment by GiantSeaweed** [comment]
I have resolved the conflict. Thanks!

**Pull request merged**